terms/collections.md

```diff
@@ -1,7 +1,12 @@
 # Semantics
 
-Agreed upon, often hierarchical, [vocabulary](change-requests.md).
+Agreed upon, often hierarchical, set of terms and structures to descibe data. This includes [vocabulary](vocabulary.md), [schema](schema.md), [taxonomies][taxonomy.md], and [ontologies][ontology.md]
 
```

Comment:
I think the best thing to do would be stick to what we agreed in the metadata challenge for this.

Schema, taxonomies and ontologies *use* semantics, but this change suggests that they *are* semantics, breaking self-consistency.

I see what you're doing though; you're responding to our own definition "often hierarchical", because how could a vocabulary be hierarchical without it being a taxonomy?!

I've realised there's a sentence that I didn't bring over from the original google doc when I was transcribing, that I think might help clear this up. I'll apply that and make a note to revisit this next metadata challenge, if that's OK with you @kilojoules